packages/headless/src/controllers/commerce/core/breadcrumb-manager/headless-core-breadcrumb-manager.test.ts

```diff
@@ -1,6 +1,14 @@
 import {Action} from '@reduxjs/toolkit';
 import {stateKey} from '../../../../app/state-key';
-import {deselectAllBreadcrumbs} from '../../../../features/breadcrumb/breadcrumb-actions';
+import {
+  clearAllCoreFacets,
```

Comment:
I renamed the commerce-specific action from `deselectAllBreadcrumbs` to `clearAllCoreFacets` because it affects all facets and will also be dispatchable from the facet generator controller.